src/Runner.Listener/JobDispatcher.cs

```diff
@@ -422,6 +422,10 @@ private async Task RunAsync(Pipelines.AgentJobRequestMessage message, string orc
                                             lock (_outputLock)
                                             {
                                                 workerOutput.Add(stdout.Data);
+                                                if (StringUtil.ConvertToBoolean(Environment.GetEnvironmentVariable(Constants.Variables.Agent.PrintLogToStdout)))
```

Comment:
store the result of the `StringUtil.ConvertToBoolean(Environment.GetEnvironmentVariable(Constants.Variables.Agent.PrintLogToStdout))` in to a variable outside of the `using`, so we don't need to read env on every line we get from the worker.